processor/src/system/mod.rs

```diff
@@ -21,6 +21,7 @@ mod tests;
 // address in a given context, regardless of which memory segment it belongs to.
 
 /// Memory addresses for procedure locals start at 2^30.
+/// DEPRECATED: use FMP_INIT_VALUE instead.
 pub const FMP_MIN: u64 = 2_u64.pow(30);
```

Comment:
Should we just remove?

Reply:
I left it there because some tests rely on it, and ultimately is more cleanly done when removing the `fmp` column in the next PR. i.e. This is going away very shortly.